docs/en-US/component/statistic.md

```diff
@@ -0,0 +1,77 @@
+
+# statistic
+
+Display statistics.
+
+## Basic usage
+
+:::demo To highlight a number or a group of numbers, such as statistical value, amount, and ranking, you can add elements such as icon and unit before and after the number and title.
+
+statistic/num
+
+:::
+
+## Countdown
+
+:::demo Set `timeIndices`Start the countdown. Countdown component, support English and Chinese countdown, support to add other components control countdown.
+
+statistic/countdown
+
+:::
+
+## Card usage
+
+:::demo Card usage display, can be freely combined
+
+statistic/card
+
+:::
+
+### Statistic Attributes
+
+| Attribute         | Description                    | Type             | Accepted Values | Default |
+| ----------------- | ------------------------------ | ---------------- | --------------- | ------- |
+| value             | Numerical content              | string \| number | -               | -       |
+| decimal-separator | Setting the decimal point      | string           | -               | .       |
+| format            | Custom numerical presentation  | string           | -               | -       |
+| group-separator   | Sets the thousandth identifier | string           | -               | ,       |
+| precision         | numerical precision            | number           | -               | 0       |
+| prefix            | Sets the prefix of a number    | string \| v-slot | -               | -       |
+| suffix            | Sets the suffix of a number    | string \| v-slot | -               | -       |
+| title             | Numeric titles                 | string \| v-slot | -               | -       |
+| value-style       | Styles numeric values          | style            | -               | -       |
+| rate              | Set the ratio                  | number           | -               | 1000    |
+
+### Statistic Slots
+
+| Name      | Description                 |
+| --------- | --------------------------- |
+| prefix    | Numeric prefix              |
+| suffix    | Suffixes for numeric values |
+| formatter | Numerical content           |
+| title     | Numeric titles              |
+
+### Statistic.Countdown Attributes
+
+| Attribute    | Description                              | Type    | Options     | Default    |
+| ------------ | ---------------------------------------- | ------- | ----------- | ---------- |
+| time-indices | Whether to enable the countdown function | boolean | true\|false | false      |
+| value        | Required value, enter the bound value    | string  | —           | —          |
+| format       | Formatting the countdown display         | string  | —           | 'HH:mm:ss' |
+
+### Statistic.Countdown Events
+
+| Method | Description                                | Parameters       |
+| ------ | ------------------------------------------ | ---------------- |
+| change | Enable in the 'countdown' function         | (value: Date)    |
+| finish | Launched after the 'countdown' is complete | (value: boolean) |
```

Comment:
## Suggest

※ this is a future consideration, so you can do it later.

We are working on improving the documentation a bit better. Please refer to the link below and update!

https://github.com/element-plus/element-plus/pull/10680/files

Reply:
The finish and change capabilities are included; now do you want to remove them?🧐

Reply:
Or do you mean I should change the instructions in the future, Only about statistic.md

Reply:
I'm sorry for the confusion.

## Example

```ts

// Add a comment as if it were in a link.
const props = defineProps({
  /**
   * @description layout direction for child elements
   */
  direction: {
    type: String,
  },
})
```

Reply:
Are you better now？